docs/README.md

```diff
@@ -75,6 +108,281 @@ The Sail-operator does not manage Gateways. You can deploy a gateway manually ei
 
 **Note:** The `IstioOperator` / `istioctl` example is separate from the Sail-operator. Setting `spec.components` or `spec.values.gateways` on your Sail-operator `Istio` resource **will not work**.
 
+## Update Strategy
+
+The sail-operator supports two update strategies to update the istio version of the control plane: `InPlace` and `RevisionBased`. The default strategy is `InPlace`.
+
+### InPlace
+When the `InPlace` strategy is used, the existing Istio control plane is updated in-place. The workloads therefore don't need to be moved from one control plane instance to another.
+
+#### Example using the InPlace strategy
+
+Prerequisites:
+* sail-operator needs to be installed.
+
+Steps:
+1. Create the `istio-system` and the `istio-cni` namespaces.
+
+    ```bash
+    kubectl create namespace istio-system
+    kubectl create namespace istio-cni
+    ```
+
+2. Create the `Istio-cni` resource
```

Comment:
```suggestion
2. Create the `IstioCni` resource
```

Reply:
But, I think we should omit `IstioCNI` here altogether. IstioCNI is not needed and mentioning it here only makes the important (upgrade) stuff less visible.

Reply:
I deleted the section, we are going to skip mentioning the istioCNI resource